charts/plugin-site/values.yaml

```diff
@@ -17,43 +51,36 @@ service:
   type: ClusterIP
 
 # Please define ingress settings into environment variable
-#ingress:
-#  enabled: true
+ingress:
+  enabled: disabled
```

Comment:
false rather than disabled? It’s checking for a Boolean but if it’s not true it’s ignored

Reply:
oops

I mean honestly its never going to be false, but yea, i'll fix

Reply:
Only reason would be someone accessing it on NodePort because they can’t get ingress working locally I think, and even then it causes no harm not to have it enables

Reply:
How about just enabling it?